GreyMatteOr/Crate
# Survey page

### This PRs additions:

This PR covers establishing the file structure for the survey 
Also builds out the state and actions for the survey page. 
This can be seen in the web/src/modules/survey/api/state.js and web/src/modules/survey/api/actions.js
This also sets the component survey
 
### Reviewer starting point: 

web/src/modules/survey/api/state.js

 
### How to manually test:  

start both the back end in the API file and the front end in the web file 
On the web page go to 
http://localhost:3000/survey
the tester should see a page that displays the words 

"Pick the items you really like!
You can choose as many as you like. Please choose at least one."

### background context:  

 This was built to start the connection between the subscription and the survey, this mainly focuses on the survey 

### Relevant tickets: 

Closes: #11 
In Progress: #10
 
### Screenshots:

not at this time
 
### Questions: 

none at this time
